examples/large_models/inferentia2/llama/continuous_batching/llama3-model-config.yaml

```diff
@@ -0,0 +1,20 @@
+minWorkers: 1
+maxWorkers: 1
+maxBatchDelay: 0
+batchSize: 8
+responseTimeout: 10800
+jobQueueSize: 500
+continuousBatching: true
+
+handler:
+    model_path: "model/models--meta-llama--Meta-Llama-3-70B/snapshots/b33784c5adf6e4b1a60d041da74e83fd438d67cd"
+    model_checkpoint_dir: "llama-3-70b-split"
+    model_module_prefix: "transformers_neuronx"
+    model_class_name: "llama.model.LlamaForSampling"
+    # see https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/torch/transformers-neuronx/index.html#known-issues-and-limitations for llama2-13b
+    # neuron_cc_flag: "-O1 --model-type=transformer --enable-mixed-precision-accumulation --enable-saturate-infinity"
+    amp: "bf16"
+    tp_degree: 24
+    max_length: 256
+    max_new_tokens: 50
+    batch_size: 8
```

Comment:
Was paged attention supported in inferentia? Does batch size of 8 give enough flexibility in that case? Would be good to discuss this in the documentation.

Reply:
Will run benchmark and update batch size.